framework/src/main/java/org/checkerframework/common/basetype/BaseTypeChecker.java

```diff
@@ -757,7 +762,9 @@ public void typeProcessingOver() {
         for (BaseTypeChecker checker : getSubcheckers()) {
             checker.typeProcessingOver();
         }
-
+        if (javacErrored && parentChecker == null && !hasOption("noWarnOnTypeCheckingHalt")) {
+            messager.printMessage(Kind.WARNING, "Javac errored; type checking halted.");
```

Comment:
This message makes it sound like javac is buggy.  I think the point is that the program is not valid Java:  normal compilation without pluggable type-checking didn't succeed.  Please clarify the message.  Text like "javac issued an error" might be good, though from the user perspective javac also issues the pluggable type-checking errors, so the message should probably be even more explicit or detailed than that.  Another clarification is that *pluggable* type checking halted.

Also, please clarify what is the consequence to users, or why they care.  Might there be `.class` files generated that didn't get type-checked?  If javac erred, wouldn't users see other warning messages?